modules/machine-edge-pool-review-nodes.adoc

```diff
@@ -0,0 +1,56 @@
+
+:_content-type: PROCEDURE
+[id="machine-edge-pool-review-nodes_{context}"]
+= Verifying nodes that were created with edge compute pool
+
+After your install a cluster that uses AWS Local Zones, check the status of the machine that was created by the machine set manifests created at install time.
+
+. To check the machine sets created from the subnet you added to the `install-config.yaml` file, enter the following command:
++
+[source,terminal]
+----
+$ oc get machineset -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  DESIRED   CURRENT   READY   AVAILABLE   AGE
+cluster-7xw5g-edge-us-east-1-nyc-1a   1         1         1       1           3h4m
+cluster-7xw5g-worker-us-east-1a       1         1         1       1           3h4m
+cluster-7xw5g-worker-us-east-1b       1         1         1       1           3h4m
+cluster-7xw5g-worker-us-east-1c       1         1         1       1           3h4m
+----
+
+. To check the machines that were created from the machine sets, enter the following command:
```

Comment:
Change to: ", run the following command:"

Reply:
@lpettyjo Changing, but the repo guidelines show both "run" and "enter." Is there a reason to use one over the other in this context?

```suggestion
. To check the machines that were created from the machine sets, run the following command:
```

Reply:
@lpettyjo Changing, but the repo guidelines show both "run" and "enter." Is there a reason to use one over the other in this context?

```suggestion
. To check the machines that were created from the machine sets, run the following command:
```

Reply:
Sigh, GitHub.